README.md

```diff
@@ -63,7 +63,7 @@ Linux, Apple OSX (Intel and M1)
 ## Requirements
 
 - Apache Spark 3.2, 3.3, or 3.4
-- JDK 8 and up
+- JDK 11 and up (JDK 8 should be supported, but development is on JDK 11)
```

Comment:
I think we should say that we only support 8, 11, and 17 rather than say JDK 11 and up. We do not support JDK 21 for example (neither does Spark 3.x).

I think we should recommend 11 for now because Spark 3.2 does not support JDK 17